app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx

```diff
@@ -8,7 +8,7 @@ interface IState {
 }
 
 export class CustomHeaderToggleBox extends React.PureComponent<IPropsCustomHeader, IState> {
-    htmlId = nextId();
+    htmlId = "togglebox-" + nextId();
```

Comment:
"togglebox" could also be passed as an argument to `nextId`. It doesn't change much, just a bit cleaner

Reply:
I get now what you mean. It is cleaner, I will change it